src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/InputStepExecution.java

```diff
@@ -71,7 +71,7 @@ public boolean start() throws Exception {
         node.addAction(new PauseAction("Input"));
 
         String baseUrl = '/' + run.getUrl() + getPauseAction().getUrlName() + '/';
-        if (input.getParameters().isEmpty()) {
+        if (input.getParameters().isEmpty() && (input.getSubmitterParameter() == null || input.getSubmitterParameter().isEmpty())) {
```

Comment:
Thanks for let me know the 
https://issues.jenkins-ci.org/browse/JENKINS-40594, @jglick 
Yes, this change could fix the JENKINS-40594.